src/SignalR/clients/ts/FunctionalTests/ts/ConnectionTests.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-// This code uses a lot of `.then` instead of `await` and TSLint doesn't like it.
-// tslint:disable:no-floating-promises
+// This code uses a lot of `.then` instead of `await` and ESLint doesn't like it.
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
It complains because of the `connection.stop()` we have, for example, at line 32. 

```
connection.onreceive = (data: any) => {
	if (data === message) {
		await connection.stop();
	}
};
```

Reply:
Done.